### PR TITLE
Fix typo in README

### PR DESCRIPTION
**Pull request by WEremite**
Fix typo and add information about myself for kottans front-end course

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and [git](https://github.com/Kottans/git-course) courses.
 You can try sending pull-requests to this repository proposing a change.
 The change can be anything you like, for example: fixing a typo in README.md. Consider this phrase, for example:
 
-*This was an intentionaly mispelld phrase*
+*This was an intentionaly misspelled phrase*
```
Comment (miroslav-kolomiets) [inline README.md:13]: Hi! But it's not typo, it's an example ;)

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Good job! But revert the example

**Review by WEremite — COMMENTED** [review]
Revert changes

File: README.md [hunk 2]
```diff
@@ -1472,3 +1472,10 @@ Thanks for the course and good luck to students! \
 ## Hello everyone!
 ### I'm [Vlad](https://github.com/MelmQuty) and I didn't find any typo in ur README(maybe because it's perfect:rocket:) but I'm gonna show your my cat.He is a little impudent.:octocat:
 ![cat](https://raw.githubusercontent.com/MelmQuty/cat/master/cat.jpg)
+
+***
+## Hi guys!
+### My names is [Alex](https://github.com/WEremite)
+Fixed a special typo in this file.
```
Comment (Temu4) [inline README.md:1479]: Good job! 
But I guess this phase has not sense. ⬆️

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Temu4) [reply README.md:1479]: Also sync latest changes from original repo and fix conflicts.

File: README.md [hunk 3]
```diff
@@ -1476,8 +1476,6 @@ Thanks for the course and good luck to students! \
 ### I'm [Vlad](https://github.com/MelmQuty) and I didn't find any typo in ur README(maybe because it's perfect:rocket:) but I'm gonna show your my cat.He is a little impudent.:octocat:
 ![cat](https://raw.githubusercontent.com/MelmQuty/cat/master/cat.jpg)
 
----
```
Comment (Temu4) [inline README.md:1479]: You shouldn't change other's code.

**Review by Temu4 — CHANGES_REQUESTED** [review]
Also can you fix conflicts?

File: README.md [hunk 4]
```diff
@@ -1476,8 +1476,6 @@ Thanks for the course and good luck to students! \
...
-
```
Comment (Temu4) [inline README.md:1480]: Please recover these lines too

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (WEremite) [reply README.md:1480]: There is someone's code on lines 1479-1480. Should I move it down and add those lines?

**Comment by babayK0** [comment]
@WEremite resolve conflicts please

**Comment by WEremite** [comment]
> 
> 
> @WEremite resolve conflicts please

Done.

File: README.md [hunk 5]
```diff
@@ -1494,13 +1494,15 @@ Thanks for the course and good luck to students! \
 ### I'm [Vlad](https://github.com/MelmQuty) and I didn't find any typo in ur README(maybe because it's perfect:rocket:) but I'm gonna show your my cat.He is a little impudent.:octocat:
 ![cat](https://raw.githubusercontent.com/MelmQuty/cat/master/cat.jpg)
 
+
```
Comment (babayK0) [inline README.md:1497]: Redundant empty line

**Review by babayK0 — CHANGES_REQUESTED** [review]

**Review by babayK0 — APPROVED** [review]

**Pull request merged**